lib/Checks.groovy

```diff
@@ -1,9 +1,29 @@
+import org.yaml.snakeyaml.Yaml
+
 /*
  * This file holds several functions used to perform standard checks for the nf-core pipeline template.
  */
 
 class Checks {
 
+    static void check_conda_channels() {
```

Comment:
```suggestion
    static void check_conda_channels(log) {
```

Reply:
Maybe we pass the `log` to this function too to render the message using `log.warn` / `log.error`?